lib/game.ex

```diff
@@ -15,4 +16,29 @@ defmodule TicTacToe.Game do
     Map.replace!(game, :players, Game.get_players(input))
   end
 
+  def add_board(game) do
+    Map.replace!(game, :board, Board.new(3))
+  end
+
+  def make_move(game, move, input \\ CLI) do
+    [ current_player | _tail ] = game.players
```

Comment:
`[ current_player | _tail ] = game.players` has been referenced twice in 2 functions now, and it's tied to this specific data structure implementation. Maybe we want to decouple that a bit more

Reply:
```Elixir
 def current_player(game) do
    Enum.at(game.players, 0)
 end
```
[I added this function to return the current player](https://github.com/digitopolis/tic-tac-toe-elixir/pull/4/commits/ddc503028e2f89551e4411c778d2b90837c0b886) I also have a function to switch between the players (just reversing that array each time, so the player at index 0 is always the current player) - do you think this is a good way to handle it?